packages/wallet-sdk/src/CoinbaseWalletSDK.ts

```diff
@@ -23,6 +23,7 @@ export class CoinbaseWalletSDK {
       appName: metadata.appName || 'Dapp',
       appLogoUrl: metadata.appLogoUrl || getFavicon(),
       appChainIds: metadata.appChainIds || [],
+      appDeeplinkUrl: null,
```

Comment:
if this is mobile only, let's provide default value for web apps and avoid any breaking changes for them.

Reply:
Should we create a `CoinbaseWalletSDK.native.ts` class? This class has some methods that are incompatible with RN like `getFavicon()` and the `window` calls in `makeWeb3Provider`

Reply:
@bangtoven yup yup, currently, when appDeeplinkUrl is null, it makes no difference for web app's case, and SDK still takes Partial<AppMetadata>, so for web dev, there should be no difference